critwall/src/main/java/org/opencadc/critwall/FileSyncJob.java

```diff
@@ -138,6 +138,9 @@
     private final Subject subject;
     private final Subject anonSubject = AuthenticationUtil.getAnonSubject();
     
+    private String downloadURL;
```

Comment:
Should be able to remove `downloadURL` member var after other requested changes.

Reply:
Yes, removed.